src/tests/functional/shared_test_classes/src/single_layer/dft.cpp

```diff
@@ -5,6 +5,18 @@
 #include "shared_test_classes/single_layer/dft.hpp"
 
 namespace LayerTestsDefinitions {
+static bool invalidTestSuffixChar(char c) {
+    switch (c) {
+    case '.':
+//    case '-':
+//    case '=':
+//    case '(':
+//    case ')':
```

Comment:
should it be removed?

Reply:
technically those all are incorrect symbols, but they don't break eclipse parser. maybe we need some global decision